bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,11 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if not all(
+            revision.contains(issue) for issue in issues if issue.is_publishable()
+        ):
+            comment += "(defects might be in the parent stack)\n"
```

Comment:
We should only add the comment when there actually is a parent revision in the stack, otherwise it'll be confusing.

Reply:
How can I check that parent revision is in the stack?
Instead, I checked issue been published through `issue.is_publishable()` function.
Could you give me directions?

Reply:
> How can I check that parent revision is in the stack?

The revision is built in bot/code_review_bot/revisions.py. Maybe the stack information could be available in the result from this call https://github.com/mozilla/code-review/blob/50ea2f7b782fb7cd81d3cc54d82d2aa4611971b8/bot/code_review_bot/revisions.py#L175, you'll need to check.

>  Instead, I checked issue been published through issue.is_publishable() function.

We should do both.

Reply:
Hello, @marco-c 
Sorry for late. I try to look at `revision` contents to check parent revision.
But, When I look at `revision`, I couldn't check parent revision.
Please, refer to the content.

- revision values
![code_review_revision_contents](https://user-images.githubusercontent.com/29684524/88896187-72b1ea80-d284-11ea-9f19-d5271812e2d0.png)